app/src/main/java/com/pajato/android/gamechat/exp/CheckersFragment.java

```diff
@@ -144,8 +146,10 @@ protected void createExperience(final Context context, final List<Account> playe
         List<Player> players = getDefaultPlayers(context, playerAccounts);
         String name1 = players.get(0).name;
         String name2 = players.get(1).name;
+
         long tstamp = new Date().getTime();
-        String name = String.format(Locale.US, "%s vs %s on %s", name1, name2, tstamp);
+        String name = String.format(Locale.US, "%s vs %s on %s", name1, name2,
```

Comment:
IMHO, code reads better when statements don't cross lines so this looks better to me as two statements on one line each rather than one statement spread over two lines.  A nit, indeed.